script.js

```diff
@@ -36,11 +36,12 @@ function generateQuote() {
     });
   tweetBtn.addEventListener("click", shareTweet);
 
-  shareTweet();
-
   function shareTweet(event) {
+    var quoteText = quote.innerText;
+    var quoteAuthor = author.innerText;
+
```

Comment:
Retrieve `quoteText` and `quoteAuthor` from the respective element's `innerText`.